Makefile

```diff
@@ -0,0 +1,86 @@
+.DEFAULT_GOAL := spec
+
+
+# builds for the current platform
+build:
+	go install
+
+# makes a new binary release
+build-release: cross-compile
+	package/debian/make_deb.sh
+
+# builds the binary for all platforms
+cross-compile:
+	go get github.com/mitchellh/gox
+	timestamp=$(TZ=UTC date -u '+%Y-%m-%dT%H:%M:%SZ')
+	sha=$(git rev-parse HEAD)
+	gox -ldflags "-X github.com/Originate/git-town/cmd.Version=$TRAVIS_TAG -X github.com/Originate/git-town/cmd.BuildTime=$timestamp) -X github.com/Originate/git-town/cmd.GitHash=$sha" \
+			-output "dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
+
+# runs the feature tests
+cuke: build
+	bundle exec parallel_cucumber $(DIR)
+DIR = $(if $(dir),$(dir),"features")
+
+# deploys the website
+deploy:
+	git checkout gh-pages
+	git pull
+	git checkout master
+	git pull --rebase
+	harp compile website/ _www
+	git checkout gh-pages
+	cp -r _www/* .
+	rm -rf _www
+	git add -A
+	git commit
+	git push
+	git checkout master
+
+# fixes all issues in all languages
```

Comment:
This sounds like it does all the programming for us! Haha. How about `auto-fix lint issues in all languages`? And for the following sections: `auto-fixes all Cucumber lint issues` and similar.

Reply:
I like it!